core/src/main/java/org/apache/spark/SparkFirehoseListener.java

```diff
@@ -34,12 +34,12 @@ public void onEvent(SparkListenerEvent event) { }
 
     @Override
     public final void onStageCompleted(SparkListenerStageCompleted stageCompleted) {
-        onEvent(stageCompleted);
+      onEvent(stageCompleted);
```

Comment:
Ah, sure. Thank you. Let me address the comments and compare this to your work too.